apps/web/src/modules/auction/hooks/useAuctionEvents.ts

```diff
@@ -43,7 +42,7 @@ export const useAuctionEvents = ({
       )
 
       await mutate([SWR_KEYS.AUCTION_BIDS, chainId, auction, tokenId], () =>
-        getBids(chainId, auction as string, tokenId)
+        getBids(chainId, collection, tokenId!.toString())
```

Comment:
Curious why you have to use the '!' assertions.

Reply:
switched to throw an error instead